server/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -374,12 +404,17 @@ private DivTag blockEditPanel(
                 ReferenceClasses.OPEN_QUESTION_BANK_BUTTON,
                 "my-4");
 
-    return div()
-        .withClasses("w-7/12", "py-6", "px-4")
-        .with(blockInfoDisplay, buttons, predicateDisplay, programQuestions, addQuestion);
+    DivTag div =
+        div()
+            .withClasses("w-7/12", "py-6", "px-4")
+            .with(blockInfoDisplay, buttons, visibilityPredicateDisplay);
+    if (maybeEligibilityPredicateDisplay.isPresent()) {
+      div = div.with(maybeEligibilityPredicateDisplay.get());
```

Comment:
suggestion (blocking): reassigning the div is unnecessary, `with` mutates it

Reply:
done